src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo,
+  isAdding?: boolean,
+  handleDelete?: (event: React.MouseEvent<HTMLButtonElement>) => void,
+  deletedId?: number[] | null,
+}
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  isAdding,
+  handleDelete,
+  deletedId,
+}) => {
+  const { title, id } = todo;
+
+  return (
+    <div data-cy="Todo" className="todo item-enter-done">
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">{title}</span>
+      <button
+        name={`${id}`}
```

Comment:
Check if it'll work
```suggestion
        name={id}
```

Reply:
No, its doesn work, it requires string..